malduck/extractor/extractor.py

```diff
@@ -352,6 +354,15 @@ def push_config(self, config):
         """
         return self.parent.push_config(config, self)
 
+    def push_file(self, data: bytes, filename=''):
+        """
+        Push file to files object
+        """
+        self.files[sha256(data).hexdigest()] = {
```

Comment:
Looking at your changes (especially at `collected_files` method), I see that you meant to collect files on parent level (ProcmemExtractManager), so they're accessible for other Extractors working on the same procmem.
```suggestion
        self.parent.files[sha256(data).hexdigest()] = {
```